src/compiler/builder.ts

```diff
@@ -62,9 +62,9 @@ namespace ts {
          */
         outSignature?: string;
         /**
-         * Time when d.ts was modified
+         * Name of the file whose dts was the latest to change
          */
-        dtsChangeTime: number | undefined;
+        dtsChangeFile: string | undefined;

```

Comment:
The JSDoc clears it up, but

```suggestion
        latestChangedDtsFile: string | undefined;
```

makes it clear from the name. (Would definitely have to look it up with its current name.)